src/lib/types.ts

```diff
@@ -80,6 +80,10 @@ export type GoTrueClientOptions = {
    * @experimental
    */
   lock?: LockFunc
+  /**
+   * Set to "true" if there is a custom authorization header
```

Comment:
```suggestion
   * Set to "true" if there is a custom authorization header set globally.
   * @experimental
```

So it's not an API people think it's theirs to use.